src/xGadgetron/cGadgetron/include/sirf/Gadgetron/cgadgetron.h

```diff
@@ -58,6 +58,8 @@ extern "C" {
 	void* cGT_setAcquisitionModelParameter
 		(void* ptr_am, const char* name, const void* ptr);
 	void* cGT_AcquisitionModelParameter(void* ptr_am, const char* name);
+	void* cGT_setCSParameter(void* ptr, const char* par, const void* val);
+	void* cGT_setAcquisitionParameter(void* ptr, const char* param_name, const void* val);
```

Comment:
not needed - Python interface calls `cGT_setParameter` (which calls one of these), see `Gadgetron_params.py`.

Reply:
Deleted the declarations